mapbox/libandroid-telemetry/src/main/java/com/mapbox/services/android/telemetry/constants/TelemetryConstants.java

```diff
@@ -10,7 +10,8 @@
 
 public class TelemetryConstants {
 
-  public static final String OPERATING_SYSTEM = "Android - " + Build.VERSION.RELEASE;
+  public static final String PLATFORM = "Android";
+  public static final String OPERATING_SYSTEM = PLATFORM + " - " + Build.VERSION.RELEASE;
```

Comment:
@ericrwolfe This has been the value the library has been setting all along (this PR doesn't change that). Happy to change the formatting upon @mick confirmation to make sure the backend pipeline doesn't get affected.